WordPress/src/test/java/org/wordpress/android/ui/mysite/items/listitem/SiteListItemBuilderTest.kt

```diff
@@ -4,6 +4,7 @@ import com.nhaarman.mockitokotlin2.whenever
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.Before
 import org.junit.Test
+import org.junit.Ignore;
```

Comment:
We need to remove the semicolon here to fix ktlint error.

Reply:
🤦 Oops, thanks for spotting! Will remove.